vhdbuilder/packer/components.json

```diff
@@ -87,7 +87,7 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-npm:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.4.29"
+        "v1.4.32"
```

Comment:
Probably want to keep `v1.4.29` in there for a while too, and remove only once AKS has stopped pulling it in all regions.

Reply:
ok, updated